### PR TITLE
Added json output flag to Disk List

### PR DESCRIPTION
**Pull request by UtkarshBhatthere**
```
$ sudo microceph disk list --json
{"ConfiguredDisks":[{"osd":1,"path":"/dev/loop5","location":"juju-3b5009-0"}],"UnpartitionedDisks":[]}
```

### CODE REVIEW

**Review by sabaini — CHANGES_REQUESTED** [review]
Hey, lgtm in general!
It would be great if we could have a test (either unit or functional) for this though

**Comment by UtkarshBhatthere** [comment]
Ack @sabaini, will add a UT.

File: microceph/cmd/microceph/init.go [hunk 1]
```diff
@@ -177,3 +178,13 @@ func (c *cmdInit) Run(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func printLocalDisks(cli *microCli.Client) error {
+	// List unpartitioned disks.
+	availableDisks, err := getUnpartitionedDisks(cli)
+	if err != nil {
+		return fmt.Errorf("internal error: unable to fetch unpartitoned disks: %w", err)
```
Comment (sabaini) [inline microceph/cmd/microceph/init.go:186]: typo: "unpartitioned"

**Review by sabaini — CHANGES_REQUESTED** [review]
Hey, one small typo in an error message, otherwise lgtm
Just a note for future reference on commit structure, it would be preferable from a to keep commits focused. The last commit message here describes adding a functional test, it does a bit more than that though :-)

File: microceph/cmd/microceph/init.go (hunk 1 above)
Comment (UtkarshBhatthere) [reply microceph/cmd/microceph/init.go]: ack

**Comment by UtkarshBhatthere** [comment]
@sabaini note taken! 😄

**Review by sabaini — APPROVED** [review]
cheers

**Pull request merged**